File: src/main/java/com/oyosite/ticon/specutils/config/CommonConfig.java
package com.oyosite.ticon.specutils.config;
;
import com.oyosite.ticon.specutils.SpectralUtilities;
import me.shedaniel.autoconfig.ConfigData;
import me.shedaniel.autoconfig.annotation.Config;

@Config(name = SpectralUtilities.MOD_ID)
public class CommonConfig implements ConfigData {
    
    public boolean jadeVinesDropJadeJellyWhenRevived = true;
    
}

File: src/main/kotlin/com/oyosite/ticon/specutils/config/CommonConfig.kt
package com.oyosite.ticon.specutils.config

import com.oyosite.ticon.specutils.SpectralUtilities
import me.shedaniel.autoconfig.ConfigData
import me.shedaniel.autoconfig.annotation.Config

@Config(name = SpectralUtilities.MOD_ID)
class CommonConfig : ConfigData {
    @JvmField
    var jadeVinesDropJadeJellyWhenRevived: Boolean = true
    @JvmField
    var AuxiliaryInkSupplierRange: Int = 3
}